### DISCUSSION TITLE
enable_point_pick()

### DISCUSSION THREAD

**Discussion by fire8flag** [pyvista/pyvista#4177] [Q&A]
enable_cell_pick() can draw a rectangular selection cell by pressing "r", but it can't select the point, is there any way to display the point inside the rectangular selection rectangle like enable_cell_pick()?

**Comment by fire8flag** [comment]
PLEASE HELP!

**Comment by banesullivan** [comment]
I am currently refactoring much of the picking code and plan to open a PR with the changes soon. It will include better point, cell, face, and edge picking.

**Reply by fire8flag** [reply]
Thank you for all your hard work in making such a great toolset available to us. Now my bijibij is more urgent to select the points inside the rectangle and return the coordinates at once, please niniy do you have any good suggestion?

**Reply by banesullivan** [reply]
Please see #4254 for picking improvements.

>  is there any way to display the point inside the rectangular selection rectangle like enable_cell_pick()?

Simply have the callback extract the points from the selected cells
